native-emulation/install.sh

```diff
@@ -13,7 +13,7 @@ readonly PTB_FILE="${TMP_DIR}/${PTB}"
 # commit hash to use for the https://github.com/dhruvvyas90/qemu-rpi-kernel/ repo:
 readonly COMMIT_HASH='c522bff346bb7401ad0b979778c23be52089618f'
 
-readonly IMAGE_URL="https://downloads.raspberrypi.org/raspbian_lite/images/raspbian_lite-2020-02-14/${IMAGE}.zip"
+readonly IMAGE_URL="https://downloads.raspberrypi.org/raspios_lite_armhf/images/raspios_lite_armhf-2020-08-24/${IMAGE}.zip"
 readonly KERNEL_URL="https://github.com/dhruvvyas90/qemu-rpi-kernel/blob/${COMMIT_HASH}/native-emulation/5.4.51%20kernels/${KERNEL}?raw=true"
```

Comment:
The 2020-02-13 kernel is a V4 kernel, and the 2020-08-24 release matches the 5.4.51 kernel version (KERNEL_URL).